.github/workflows/publish.yml

```diff
@@ -31,6 +31,9 @@ jobs:
       with:
         node-version: 16
 
+    - name: Configure npm
+      run: npm config set //wombat-dressing-room.appspot.com/:_authToken=$NODE_AUTH_TOKEN
```

Comment:
Honestly not sure if it's supposed to be `$NODE_AUTH_TOKEN` or `${NODE_AUTH_TOKEN}`

Reply:
I could be wrong but I think you would need the braces if this were in a `.npmrc` but not with this command so you're good.